src/components/DataCollector/createConfigValidators.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+  required,
+  validDomain,
+  allValidate,
+  unique
+} from "../../utils/configValidators";
+
+export default (propertyIdValues, orgIdValues) => {
```

Comment:
Is there any benefit to these parameters rather than just creating the arrays inside the function?

Reply:
It makes testing easier.

Reply:
How? It seems we're passing in `[], []` everywhere.

Reply:
Correct, but each test is isolated because they all use different array instances.

Reply:
That would also be the case if we just created the arrays inside this function, no?

Reply:
Oh, I see we could just have empty arrays in the function.  Yes that would work.